configs/mocha-config-compass/register/dns-config-register.js

```diff
@@ -0,0 +1,2 @@
+const dns = require('dns');
+dns.setDefaultResultOrder('ipv4first');
```

Comment:
This should come with some sort of explanation and a TODO comment about removing this as soon as we can